### PR TITLE
Add sustainability extension, update extensions versions

### PR DESCRIPTION
**Pull request by yolile**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (jpmckinney) [inline manage.py]: Why is this needed? If the virtualenv has python 3, the original will work.

File: extension_versions.csv [hunk 2]
```diff
@@ -81,10 +81,22 @@ signatories,,master,https://raw.githubusercontent.com/open-contracting-extension
...
+lots,2018-01-30,v1.1.3,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1.3/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1.3
+lots,2017-08-07,v1.1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1.1
+lots,2017-05-09,v1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1
+process_title,2017-08-07,v1.1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_process_title_extension/v1.1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_process_title_extension/zipball/v1.1.1
+process_title,2017-05-09,v1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_process_title_extension/v1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_process_title_extension/zipball/v1.1
```
Comment (jpmckinney) [inline extension_versions.csv:102]: How did these end up here? I removed these in c54a52c6243e52c8a437659b5c8b840aa09dbd2d and bfe08e9713a9882212fc394cdcbf2e23a088bcc1

File: manage.py (hunk 1 above)
Comment (jpmckinney) [reply manage.py]: Nevermind - this was reverted.

File: build/extensions.json [hunk 3]
```diff
@@ -741,6 +741,19 @@
       },
       "url": "https://raw.githubusercontent.com/open-contracting-extensions/ocds_submissionTerms_extension/master/"
     },
+    {
+      "category": "tender",
+      "core": false,
+      "description": {
+        "en": "Adds a sustainability array and a hasSustainability field to the tender and lot levels to identify if this process is related to Sustainable Public Procurement (SPP)."
```
Comment (jpmckinney) [inline build/extensions.json:748]:
```suggestion
        "en": "Adds fields to the tender and lot objects, to provide information related to Sustainable Public Procurement."
```

File: build/extensions.json [hunk 4]
```diff
@@ -741,6 +741,19 @@
...
+      },
+      "documentation_url": "https://github.com/open-contracting-extensions/ocds_sustainability_extension",
```
Comment (jpmckinney) [inline build/extensions.json:750]: I guess we update it to the explorer in a later commit?

File: extension_versions.csv (hunk 2 above)
Comment (yolile) [reply extension_versions.csv:102]: I ran `./manage.py refresh`

File: build/extensions.json (hunk 4 above)
Comment (yolile) [reply build/extensions.json:750]: Yes, I put that URL because a test fails if I leave the URL blank

File: extension_versions.csv (hunk 2 above)
Comment (jpmckinney) [reply extension_versions.csv:102]: Ah, I guess I'll add some exceptions to it.

Comment (jpmckinney) [reply extension_versions.csv:102]: For now, please delete them from the PR.

Comment (yolile) [reply extension_versions.csv]: Donw

**Comment by jpmckinney** [comment]
Build will fail until https://github.com/open-contracting-extensions/ocds_sustainability_extension/pull/2 is merged
